opentreemap/treemap/tests/__init__.py

```diff
@@ -409,7 +409,7 @@ def ecoservice_not_running():
     try:
         status = subprocess.check_output(
             ["sudo", "service", settings.ECOSERVICE_NAME, "status"])
-        return status.find('start/running') < 0
+        return status.find('active (running)') < 0
```

Comment:
The text this was looking for no longer exists because `systemd` has replaced Upstart.